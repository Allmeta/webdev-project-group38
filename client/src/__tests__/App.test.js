import React from 'react'

test('Dummy test', () => {
  expect(1).toEqual(1)
});