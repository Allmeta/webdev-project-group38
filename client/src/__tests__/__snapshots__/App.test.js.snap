// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <ul>
    <li>
      This is a test
    </li>
    <li>
      This is a second test
    </li>
  </ul>
</div>
`;
